conbench/tests/api/test_benchmarks.py

```diff
@@ -843,3 +843,50 @@ def test_neither_stats_nor_error_field_is_present(self, client):
         response = client.post(self.url, json=data)
         message = {"_schema": ["Either stats or error field is required"]}
         self.assert_400_bad_request(response, message)
+
+    def test_create_benchmark_context_missing(self, client):
+        self.authenticate(client)
+        payload = _fixtures.VALID_PAYLOAD.copy()
+        del payload["context"]
+        resp = client.post(self.url, json=payload)
+        assert resp.status_code == 400, f"unexpected response: {resp.text}"
+        assert '"context": ["Missing data for required field."]' in resp.text
+
+    def test_create_benchmark_context_empty(self, client):
+        """
+        It is an error to provide no context object (see test above). Whether
+        an empty context object is something we want to accept is debatable. As
+        part of working on https://github.com/conbench/conbench/issues/365 we
+        for now opt for accepting it, i.e. HTML template rendering must expect
+        this scenario.
+        """
+        self.authenticate(client)
+        payload = _fixtures.VALID_PAYLOAD.copy()
+        payload["run_id"] = _uuid()
+        payload["batch_id"] = _uuid()
+
+        payload["context"] = {}
+        resp = client.post(self.url, json=payload)
+        assert resp.status_code == 201, f"unexpected response: {resp.text}"
+
+        bechmark_id = resp.json["id"]
```

Comment:
nit:
```suggestion
        benchmark_id = resp.json["id"]
```
(and down below when you use it again)

Reply:
Thanks!